src/useNormalizeJsonify/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Utility helper methods specific for Sixa projects.
+ *
+ * @ignore
+ */
+import { isNonEmptyArray, jsonify } from '@sixa/wp-block-utils';
+
+/**
+ * Validate and normalize JSON string to an array.
+ *
+ * @function
+ * @since      1.5.0
+ * @param 	   {string}    value    JSON string to be normalized to an array.
+ * @return     {Array}			    Constructed JavaScript object from the given JSON string.
+ * @example
+ *
+ * const options = useNormalizeJsonify( '[\u0022Search Engine\u0022,\u0022Social Media\u0022]' );
+ *
+ * // => Array [ 'Search Engine', 'Social Media' ]
+ */
+function useNormalizeJsonify( value = '[""]' ) {
+	const jsonifyValue = jsonify( value );
+
+	return ! isNonEmptyArray( jsonifyValue ) ? [ '' ] : jsonifyValue;
```

Comment:
Would we be able to use an empty array in these instances instead of an array with a single empty string?

Whenever calling this function with a faulty or empty input, I would expect the function to return an empty array. For instance, the following operation would lead to unexpected behaviour:

```JS
const someArrayOfItems = useNormalizeJsonify( myJsonWithSomeOrNoItems );

if ( ! someArrayOfItems.length ) {
    return;
}

return (
    <ul>
        { map( someArrayOfItems, ( item ) => <li>{ item.name }</li> ) }
    <ul>
);
```

Would you be able to share an example of where this function is used?